addon/package.json

```diff
@@ -0,0 +1,123 @@
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/emberjs/ember-test-waiters",
+  "license": "MIT",
+  "contributors": [
+    "Steve Calvert <steve.calvert@gmail.com>",
+    "Robert Jackson <me@rwjblue.com>"
+  ],
+  "types": "index.d.ts",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:js": "eslint --ext ts,js .",
+    "lint:js:fix": "eslint --ext ts,js . --fix",
+    "lint:ts": "tsc -p tsconfig.json --noEmit",
+    "prepack": "npm-run-all prepack:*",
+    "prepack:d.ts": "ember ts:precompile",
+    "prepack:addon": "babel --extensions '.ts' --presets @babel/preset-typescript addon --out-dir addon --ignore '**/*.d.ts'",
+    "postpack": "ember ts:clean",
+    "start": "ember serve",
+    "test": "ember test",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "test:prod": "ember test --environment=production"
+  },
+  "dependencies": {
+    "@glimmer/env": "^0.1.7",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-typescript": "^5.0.0",
+    "ember-cli-version-checker": "^5.1.2",
+    "semver": "^7.3.5"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.14.8",
+    "@babel/preset-typescript": "^7.15.0",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.1",
+    "@types/ember": "^3.16.5",
+    "@types/ember-test-helpers": "^1.0.10",
+    "@types/ember-testing-helpers": "^0.0.4",
+    "@types/ember__test-helpers": "^2.6.1",
+    "@types/qunit": "^2.11.2",
+    "@types/rsvp": "^4.0.4",
+    "@typescript-eslint/eslint-plugin": "^4.29.2",
+    "@typescript-eslint/parser": "^4.30.0",
+    "co": "^4.6.0",
+    "ember-cli": "^4.10.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-test-loader": "^3.0.0",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-concurrency": "^2.1.2",
+    "ember-concurrency-decorators": "^2.0.3",
+    "ember-concurrency-ts": "^0.3.1",
+    "ember-decorators-polyfill": "^1.1.5",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-load-initializers": "^2.1.2",
+    "ember-maybe-import-regenerator-for-testing": "^1.0.0",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.26.1",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-try": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ember": "^7.8.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.5.1",
+    "promise-polyfill": "^8.2.0",
+    "qunit": "^2.16.0",
+    "release-it": "^14.14.3",
```

Comment:
not an action: in a follow up we may want to extract this to root level

Reply:
100% -- I have to finish https://github.com/release-it-plugins/workspaces/pull/110/ first -- thankfully we have a while before we'll want to release these changes, per: https://github.com/emberjs/ember-test-waiters/issues/458